samples/glean/src/main/java/org/mozilla/samples/glean/MainActivity.kt

```diff
@@ -13,6 +13,8 @@ import mozilla.components.service.glean.config.Configuration
 import mozilla.components.support.base.log.Log
 import mozilla.components.support.base.log.sink.AndroidLogSink
 
+import mozilla.components.service.glean.Glean
```

Comment:
Why is this needed?

Reply:
Sorry, I think that was just a merge glitch that I missed.  Removing.